docs/providers/aws/guide/functions.md

```diff
@@ -37,6 +37,7 @@ functions:
     runtime: python2.7 # optional overwrite, default is provider runtime
     memorySize: 512 # optional, in MB, default is 1024
     timeout: 10 # optional, in seconds, default is 6
+    concurrency: 5 # optional, concurrency limit for this function. By default, it is not set and AWS use Account concurrency limit
```

Comment:
we could name it `reservedConcurrency` as it is named so in CF and AWS too ?

Reply:
Sure, I will change it